src/main/java/org/folio/circulation/domain/FeeFineOwner.java

```diff
@@ -0,0 +1,38 @@
+package org.folio.circulation.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+public class FeeFineOwner {
```

Comment:
Is a fee / fine owner a user?

Reply:
Fee/fine owner is an entity defined in the mod-feesfines and configured in the UI (Settings -> Users -> Fee/fine -> Owners). As it says in the JSON schema - "Service desk (known as a fee/fine owner) is library-defined and is associated with specific fees/fines".

Reply:
Ok, thanks. I was hoping it might be a staff member